sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -59,6 +59,7 @@
   static final String ES_TYPE = "test";
   static final long NUM_DOCS_UTESTS = 400L;
   static final long NUM_DOCS_ITESTS = 50000L;
+  static final int ACCEPTABLE_EMPTY_SPLITS_PERCENTAGE = 50;
```

Comment:
You need this to be 0.5 otherwise your assertThat will be numSplits * 50 which is much greater then the number of splits produced.

Reply:
oups ... I was a bit too quick :)